proposals/privacy-threat-model.html

```diff
@@ -149,6 +150,10 @@ <h2>
         cards (e.g., business and personal) belong to the same individual, the
         user should make use of two profiles.
         </li>
+        <li>Although "private browsing mode" may help mitigate some of the
+        threats described here, we do not consider it a sufficient mitigation
+        strategy.
+        </li>
         <li>Most browsers are moving in the direction of double-keyed
         partitioning, whatever the storage mechanism.
```

Comment:
Since we're in the vicinity... I think we might need to double check the accuracy of this statement. "Double-keyed partitioning" may be true to cookies, but AFAIK, it's not yet true for other types of storage. I'm also not following why this needs to be an assumption? (sorry my memory is failing me...)

Reply:
Hi @danyao,

I had understood that there is a goal for double-keyed partitioning in other storage mechanisms as well. If the mechanisms are, in some cases, interchangeable it's not clear why one would be allowed to have more power than others. I could be wrong and so I'll ask around again.

The relevance of the comment is not obvious. Would this replacement be more appropriate:

 "We expect browsers to move in a direction where mechanisms used for cross-origin communications, such as iframes, popups, and payment hanadlers, to have those mechanisms open in a third party context by default."